### ISSUE TITLE
When running the steps provided in the guide in a Github actions CI pipeline I get fatal: not a git repository (or any of the parent directories): .git

### ISSUE THREAD

**Issue by PaulSender** [getsentry/sentry-javascript-bundler-plugins#350]
### Environment

v2.4.0

### Steps to Reproduce

1. Follow steps from `npx @sentry/wizard@latest -i sourcemaps`
2. Or same steps from manual vite specific docs

### Expected Result

I would expect there to be no error at build time

### Actual Result

At build time, I get an error: 

```
#30 [node-backend-base 25/26] RUN npm run build
#30 0.579 
#30 0.579 > frontend@0.1.0 build
#30 0.579 > vite build
#30 0.579 
**#30 1.080 fatal: not a git repository (or any of the parent directories): .git**
#30 1.086 [sentry-vite-plugin] Warning: No release name provided. Will not inject release. Please set the `release.name` option to identifiy your release.
#30 1.086 [sentry-vite-plugin] Warning: No release name provided. Will not create release. Please set the `release.name` option to identifiy your release.
#30 1.149 vite v3.2.7 building for production...
#30 1.184 [sentry-vite-plugin] Info: Sending error and performance telemetry data to Sentry. To disable telemetry, set `options.telemetry` to `false`.
#30 1.320 transforming...
#30 23.14 ✓ 3480 modules transformed.
#30 43.44 
#30 43.44 <--- Last few GCs --->
#30 43.44 
#30 43.44 [18:0x5ff6c40]    40555 ms: Mark-sweep 2033.9 (2089.8) -> 2032.1 (2088.8) MB, 1279.1 / 0.0 ms  (average mu = 0.237, current mu = 0.023) allocation failure; scavenge might not succeed
#30 43.44 [18:0x5ff6c40]    42851 ms: Mark-sweep 2041.4 (2090.2) -> 2040.0 (2119.1) MB, 2291.0 / 0.0 ms  (average mu = 0.107, current mu = 0.002) allocation failure; scavenge might not succeed
#30 43.44 
#30 43.44 
#30 43.44 <--- JS stacktrace --->
#30 43.44 
#30 43.44 FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
#30 43.45  1: 0xb7a940 node::Abort() [node]
#30 43.45  2: 0xa8e823  [node]
#30 43.45  3: 0xd5c940 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
#30 43.45  4: 0xd5cce7 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
#30 43.45  5: 0xf3a3e5  [node]
#30 43.45  6: 0xf4c8cd v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [node]
#30 43.45  7: 0xf26fce v8::internal::HeapAllocator::AllocateRawWithLightRetrySlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [node]
#30 43.45  8: 0xf28397 v8::internal::HeapAllocator::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [node]
#30 43.45  9: 0xf0956a v8::internal::Factory::NewFillerObject(int, v8::internal::AllocationAlignment, v8::internal::AllocationType, v8::internal::AllocationOrigin) [node]
#30 43.45 10: 0x12ce7af v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [node]
#30 43.45 11: 0x16fb6b9  [node]
#30 43.76 Aborted (core dumped)
#30 ERROR: executor failed running [/bin/sh -c npm run build]: exit code: 134
------
 > [node-backend-base 25/26] RUN npm run build:
43.45  3: 0xd5c940 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
43.45  4: 0xd5cce7 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
43.45  5: 0xf3a3e5  [node]
43.45  6: 0xf4c8cd v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [node]
43.45  7: 0xf26fce v8::internal::HeapAllocator::AllocateRawWithLightRetrySlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [node]
43.45  8: 0xf28397 v8::internal::HeapAllocator::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [node]
43.45  9: 0xf0956a v8::internal::Factory::NewFillerObject(int, v8::internal::AllocationAlignment, v8::internal::AllocationType, v8::internal::AllocationOrigin) [node]
43.45 10: 0x12ce7af v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [node]
43.45 11: 0x16fb6b9  [node]
43.76 Aborted (core dumped)
------
ERROR: failed to solve: executor failed running [/bin/sh -c npm run build]: exit code: 134
Error: buildx failed with: ERROR: failed to solve: executor failed running [/bin/sh -c npm run build]: exit code: 134
```

Investigation showed the same problem but in another plugin: https://github.com/ikenfin/vite-plugin-sentry. 

Other question here is which package do I use? The guide steps don't work for my workflow. I don't have a .git repo when building/deploying. What's the workaround?

**Comment by lforst** [comment]
Hi, thanks for writing in. To me, this doesn't really look like an issue with the plugin but rather looks like your machine is running out of memory.

I don't think this is something we can easily fix in the plugin as we're not doing very intensive computations (at least not more intensive than the average transpilation).

**Comment by lforst** [comment]
Likely fixed with #379 - feel free to ping us when the issue persists as soon as the new version is released.
